src/test/java/com/github/fairit/destinypro/serviceimpl/character/CharacterApiServiceTest.java

```diff
@@ -18,10 +18,10 @@
 public class CharacterApiServiceTest {
 
     @Autowired
-    private CharacterApiService characterApiService;
+    private CharacterApiServiceImpl characterApiService;
```

Comment:
use interface only i.e. `CharacterApiService`

change in all other places

Reply:
done